src/applications/dhp-connected-devices/tests/dhp-consent-flow.cypress.spec.js

```diff
@@ -1,15 +1,27 @@
 import manifest from '../manifest.json';
 
 describe(manifest.appName, () => {
-  // Skip tests in CI until the app is released.
-  // Remove this block when the app has a content page in production.
-  before(function() {
-    if (Cypress.env('CI')) this.skip();
-  });
-
   it('is accessible', () => {
```

Comment:
See the warnings on this file: "Cypress E2E tests must include at least one axeCheck call". Not sure what this means, but seems to be related to linting rules

Reply:
I saw some of the other tests including `axeCheck`, we'll work on that. Thanks for pointing that out

Reply:
Where were you seeing the warnings @yonidoronpeters

Reply:
We actually attempted to run an `axeCheck` and the accessibility issues were related to modals & components that we didn't make. A lot of the issues were around using the same ID for various HTML elements.

Reply:
@reblg seeing it in the PR under the Files changed tab
![Screen Shot 2022-03-29 at 11 51 26 AM](https://user-images.githubusercontent.com/1357047/160674396-e07ef23b-0c33-47d6-a8bf-24991bba42fa.png)